Client/src/Components/Label/index.jsx

```diff
@@ -127,11 +127,11 @@ const StatusLabel = ({ status, text, customStyles }) => {
 	const colors = {
 		up: {
 			dotColor: theme.palette.success.main,
-			bgColor: theme.palette.success./* dark */ contrastText,
+			bgColor: theme.palette.success.contrastText /* dark */,
 			borderColor: theme.palette.success.main /* light */,
```

Comment:
I had left the light and dark commented just so we know what was there before, I think we can delete these comments (not necessarily on this pr)